osu.Game/Overlays/OptionsOverlay.cs

```diff
@@ -33,10 +33,13 @@ public class OptionsOverlay : OverlayContainer
 
         private ScrollContainer scrollContainer;
         private OptionsSidebar sidebar;
+        private OptionsSidebar.SidebarButton[] sidebarButtons;

```

Comment:
I'd really prefer to keep non-reusable components as nested classes until we find a reason to reuse them. Right now nothing but OptionsSidebar should have SidebarButtons as children.